contrib/templates/cds-template-only-git-clone-job/template.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/template"
+)
+
+type TemplateOnlyGitCloneJob struct {
+	template.Common
+}
+
+func (t *TemplateOnlyGitCloneJob) Name() string {
+	return "cds-template-only-git-clone-job"
+}
+
+func (t *TemplateOnlyGitCloneJob) Description() string {
+	return `
+This template creates:
+- a build pipeline with	one stage, containing one job
+- job contains 2 steps: CDS_GitClone and a empty script.
```

Comment:
Why not use the builtin gitclone action ? It's better than the old CDS_GitClone one